packages/venia-concept/src/ServiceWorker/registerRoutes.js

```diff
@@ -19,6 +23,25 @@ import { THIRTY_DAYS, MAX_NUM_OF_IMAGES_TO_CACHE } from './defaults';
 export default function() {
     const catalogCacheHandler = createCatalogCacheHandler();
 
+    /**
+     * Never cache admin URL assets. Always go to the network.
+     *
+     * Will match URLs like:
+     * 1. venia.com/admin
+     * 2. venia.com/admin/admin
+     * 3. venia.com/backend/admin
+     * 4. venia.com/admin/store/settings
+     *
+     * Will not match URLs like:
+     * 1. venia.com/store_admin
+     * 2. venia.com/store/store_admin/settings
+     */
+    registerRoute(
+        ({ url }) =>
+            url.pathname.endsWith('/admin') || url.pathname.includes('/admin/'),
```

Comment:
The admin URL for Magento is configurable and typically randomly generated or changed due to security issues.

We also would want to avoid exposing the admin path within client side code, which may make this fix harder.

Reply:
I was under the impression that the admin URL will have `/admin` all the time but can be masked around with other paths for instance `/******/admin` or `store_backend/admin` to name a couple of examples.

@chris-brabender the problem with `.env` or other configs is, they might be hidden from the public but at the end of the day the SW has to reference those values and it will be doing it publicly. There is no way we can keep that value hidden if we choose to use it.

Reply:
Can an admin panel URL not have `/admin` in it?

Reply:
Yes https://example.com/custom323/sales/order/ could be an admin route. Does the SW have access to cookies? The backend will set an "admin" cookie that has as its path the custom admin url
![image](https://user-images.githubusercontent.com/455508/97070369-4106b080-1634-11eb-9330-0ee35153081a.png)

If that works I guess the issue would only present itself if someone went to the frontend first. A workaround for this if we don't want to solve this without changes to the graphql url resolver could be to add a special query to force the SW to go network only. 

Would you mean admin users need to use an adopted url (https://example.com/custom323/?sw_network_only=true
) to get to the login but that may be easier than explaining how to unregister the SW.